packages/block-library/src/latest-comments/index.php

```diff
@@ -43,6 +43,13 @@ function wp_latest_comments_draft_or_post_title( $post = 0 ) {
  * @return string Returns the post content with latest comments added.
  */
 function render_block_core_latest_comments( $attributes ) {
+	// Handle backward compatibility: check for old displayExcerpt attribute
+	if ( ! isset( $attributes['displayContent'] ) && isset( $attributes['displayExcerpt'] ) ) {
```

Comment:
```suggestion
	if ( isset( $attributes['displayExcerpt'] ) ) {
```

I believe this conditional statement is incorrect. That's because `$attributes['displayContent']` always exists, so this condition will never be true.